src/components/EventBanner/EventBanner.tsx

```diff
@@ -0,0 +1,79 @@
+import React from "react";
+import "./EventBanner.scss";
+import dayjs from "dayjs";
+import * as SEC from "../SEC/lib.js";
+
+interface EventBannerProps {
+	flyerSource: any;
+	altText: string;
+	title: string;
+	desc: string;
+	startDate: dayjs.Dayjs;
+	endDate: dayjs.Dayjs;
+	meetingLink: string;
+	isGbm: boolean;
+	now?: dayjs.Dayjs;
+}
+
+const EventBanner = (props: EventBannerProps): JSX.Element | null => {
+	let now: dayjs.Dayjs = props.now ? props.now : dayjs();
```

Comment:
can this be `props.now ?? dayjs()`? or do we not have that enabled

Reply:
what does that syntax mean?

Reply:
https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Nullish_coalescing_operator